.gitignore

```diff
@@ -1,5 +1,6 @@
 assets/override
 dist
+build-ios
```

Comment:
why is this needed here? once the app builds this folder should be removed entirely, and it seems to do just that in the Make command